custom-headers-with-reportviewer-rest-api-calls.md

```diff
@@ -0,0 +1,62 @@
+---
+title: How to Pass Custom Headers Along with Report Viewer's REST API Calls from Angular Application
+description: Add Custom Headers to Angular Report Viewer's REST API Calls
+type: how-to
+page_title: Pass Custom Headers Along with the Angular Report Viewer's REST API Calls
+slug: custom-headers-with-reportviewer-rest-api-calls
+position: 
+tags: 
+ticketid: 1474331
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>14.1.20.513</td>
```

Comment:
The KB is not specific for this version. In such cases, we don't put a version tag

Reply:
Thanks for the tip, I will remove the version tag.